tnr/service.py

```diff
@@ -97,7 +97,23 @@ def timespan_byname_v11(name):
             dec = resolver_data['dec_deg']
             have_coordinates = True
             break
-        
+
+    object_type=None
+    object_links=None
+    for resolver_name in 'sesameproxy.SesameProxyResolver', 'gcproxy.GCProxyResolver':
```

Comment:
can you use one `for` loop instead of 3? you'd  need to avoid `break`s, but then you have less repetetion